src/CartesianProduct.jl

```diff
@@ -180,32 +150,34 @@ Type that represents the Cartesian product of a finite number of convex sets.
 
 ### Notes
 
-- `CartesianProductArray(array::Vector{<:LazySet})` -- default constructor
+The `EmptySet` is the absorbing element for `CartesianProductArray`.
 
-- `CartesianProductArray()` -- constructor for an empty Cartesian product
+Constructors:
```

Comment:
while we are at this line, a general thought is that i find the illustration of the different constructors to be more friendly if it is given as a proper example, rather than a copy paste of the source code -- if one wants that level of detail, then it's better to just look the code, by clicking on "view source", no? (or use Julia's `@edit` macro). that is to say that for me we can delete this part and instead put some examples of the different uses (in another PR).

Reply:
I added #232.